lib/facter/pe_status_check.rb

```diff
@@ -219,6 +219,19 @@
     end
   end
 
+  chunk(:S0029) do
+    next unless PEStatusCheck.replica? || PEStatusCheck.postgres? || PEStatusCheck.primary?
+    # check if concurrnet connections to Postgres approaching 90% defined
+
+    maximum = PEStatusCheck.max_connections.to_i
+    current = PEStatusCheck.cur_connections.to_i
+    percent_used = (current / maximum.to_f) * 100
+    { S0029: percent_used >= 90 }
```

Comment:
```suggestion
    { S0029: percent_used <= 90 }
```

Should be less than

Reply:
There is conflict, hence I created new. Will modify and pull again. Thanks